tasks/client.yml

```diff
@@ -2,3 +2,44 @@
 # TODO Install clients:
 ## kubectl
 ## aws-iam-authenticator
+
+- name: "Get kubectl path"
+  shell: "which kubectl || echo 'not found'"
+  register: kubectl_path
+
+- name: Create bin directory
+  file:
+    path: /opt/k8s
+    state: directory
+
+- name: Download kubectl
+  get_url:
+    url: https://amazon-eks.s3-us-west-2.amazonaws.com/1.11.5/2018-12-06/bin/linux/amd64/kubectl
+    dest: /opt/k8s/kubectl
+    mode: 0775
+  when: '"not found" in kubectl_path.stdout'
+
+- name: Create link to kubectl
+  file:
+    src: /opt/k8s/kubectl
+    dest: /usr/local/bin/kubectl
```

Comment:
@brunomanzo  why not download it directly to its path? instead of doing a link ?